### DISCUSSION TITLE
Inline Copilot Chat window - resize?

### DISCUSSION THREAD

**Discussion by Caleb-D** [community/community#113212] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Can I resize it?  it's too small (need it to be taller)

If not, can it be enhanced in a future Copilot Chat release?

**Comment by q-danpryer** [comment]
Bump +1 on this. Just seen you can do a "diff" in the inline chat however this window is tiny and it seems like you cant resize it, making viewing the diff so hard its almost pointless.
